server/utils/bulkUploadUtils.js

```diff
@@ -0,0 +1,5 @@
+exports.createWorkerKey = (localEstablishmentId, workerId) => {
+  return ((localEstablishmentId || '') + (workerId || '')).replace(/\s/g, '');
+};
```

Comment:
This could be shorter
```suggestion
exports.createWorkerKey = (localEstablishmentId, workerId) => ((localEstablishmentId || '') + (workerId || '')).replace(/\s/g, '');
```